### PR TITLE
Update ExDB schemas for RO-3979

### PR DESCRIPTION
**Pull request by brindakv**
This PR addresses RO-3979. Updated schemas to
 - support reference sequence coverage information

Rolled back RO-4002 schema updates to improve indexing for `pdbx_initial_refinement_model`. This has been addressed by https://github.com/rcsb/py-rcsb_exdb_assets/pull/126.

### CODE REVIEW

File: dictionary_files/extensions/container_identifiers/rcsb_polymer_entity_container_identifiers.dic [hunk 1]
```diff
@@ -4,13 +4,19 @@ data_rcsb_polymer_entity_container_identifiers.dic
 _pdbx_dictionary_component.datablock_id   rcsb_polymer_entity_container_identifiers.dic
 _pdbx_dictionary_component.dictionary_component_id  rcsb_polymer_entity_container_identifiers.dic
 _pdbx_dictionary_component.title          "RCSB extension dictionary: rcsb_polymer_entity_container_identifiers.dic"
-_pdbx_dictionary_component.version        1.0.2
+_pdbx_dictionary_component.version        1.0.3
```
Comment (piehld) [inline dictionary_files/extensions/container_identifiers/rcsb_polymer_entity_container_identifiers.dic:7]: This doesn't need to match the version in `exdb-config-schema.yml`, does it? It's just a coincidence they are only 0.0.1 different, right?

```
      - NAME: pdbx_comp_model_core_polymer_entity
        VERSION: 1.0.2
```

Comment (brindakv) [reply dictionary_files/extensions/container_identifiers/rcsb_polymer_entity_container_identifiers.dic:7]: @piehld No these don't need to match.

**Review by piehld — APPROVED** [review]
LGTM

**Comment by brindakv** [comment]
Thanks @piehld

**Pull request merged**